src/index.html

```diff
@@ -13,10 +13,12 @@
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
   <script src="assets/Chart.bundle.min.js"></script>
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-
+  <!-- Meta data for search engines -->
   <meta name="description"
     content="Susper is a decentral Search Engine that uses the peer to peer system
     'YaCy' and 'Apache Solr' to crawl and index search results.">
+
+  <!-- @0x48piraj :: Basic meta data for ranking high in google search, and making large tiles in Twitter when susper.com is mentioned.  -->
```

Comment:
Please remove `@0x48piraj ::`

Reply:
And the reason behind this ?

Reply:
@0x48piraj I guess because no one else has put their names in the code.

Reply:
Nah!
Check sincerely. You'll find plenty of comments like these and with devs contact details too.

**Mantra** - *Please do some research before pointing out fingers. Because of this, many Pull requests get stagnant.*